### PR TITLE
Update NGCC SOEC example for new costing methods

### LINKED ISSUE #150: CI failures caused by ngcc_soec notebook
## Long runtimes

image

## Solver failures

image

### PR DESCRIPTION
**Pull request by bpaul4**
## Proposed changes:
- Re-runs notebook to remove warnings currently visible in GitHub version of example
- Updates costing to use new power plant costing methods
- Note: this will work once https://github.com/IDAES/idaes-pse/pull/978 is merged, and this PR should be merged ASAP afterwards since the new methods will break the existing notebook
- **The head was temporarily changed to 978 to test the example and should be reverted to main once 978 is merged.**

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Comment by bpaul4** [comment]
Until the new power plant costing methods are merged, the expected outcome is that only the NGCC SOEC example fails (due to an `ImportError`).

File: .github/workflows/main.yml
Comment (lbianchi-lbl) [inline .github/workflows/main.yml:23]: Just a reminder to switch this back to the original line before merging this PR.

**Review by lbianchi-lbl — CHANGES_REQUESTED** [review]
Nothing against the changes in the PR, this is simply a reminder to restore the `idaes-pse` pull URL before merging this PR (hopefully pushing that change should automatically dismiss this review; otherwise, feel free to ping me).

**Review by lbianchi-lbl — APPROVED** [review]

**Review by AlexNoring — APPROVED** [review]
lgtm

**Pull request merged**
